packages/twenty-front/src/modules/object-record/record-title-cell/hooks/useRecordTitleCell.tsx

```diff
@@ -95,19 +108,29 @@ export const useRecordTitleCell = () => {
         });
         set(isInlineCellInEditModeScopedState(recordTitleCellId), true);
 
-        const fieldDefinitions = objectMetadataItem.fields.map(
-          (fieldMetadataItem, index) =>
-            formatFieldMetadataItemAsColumnDefinition({
-              field: fieldMetadataItem,
-              objectMetadataItem,
-              position: index,
-            }),
-        );
+        const recordIndexFieldDefinitions = snapshot
```

Comment:
Let's not reintroduce this as it should be removed, it is already derived from objectMetadataItems which is the source of truth. 

Having those errors in Sentry is good because before they were silent, reverting the change will just make it silent again, instead we should investigate why this happens, why some users end up with undefined values.

Reply:
@lucasbordeau could you then please own and followup the Sentry issue? Thank you